Client/Frontend/Browser/Playlist/Controllers/PlaylistViewController.swift

```diff
@@ -55,12 +56,17 @@ class PlaylistViewController: UIViewController {
     private var assetStateObservers = Set<AnyCancellable>()
     private var assetLoadingStateObservers = Set<AnyCancellable>()
     
-    init(mediaPlayer: MediaPlayer, initialItem: PlaylistInfo?, initialItemPlaybackOffset: Double) {
+    private(set) weak var browser: BrowserViewController?
+    
+    init(browser: BrowserViewController?, mediaPlayer: MediaPlayer, initialItem: PlaylistInfo?, initialItemPlaybackOffset: Double) {
+        self.browser = browser
```

Comment:
We don't need to pass in `browser` anymore do we?  Shouldn't you be passing in a `PlaylistViewControllerDelegate`?